tests/playwright/src/specs/kubernetes-edit-yaml.spec.ts

```diff
@@ -0,0 +1,124 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+import { PlayYamlRuntime } from '../model/core/operations';
+import { KubernetesResourceState } from '../model/core/states';
+import { KubernetesResources } from '../model/core/types';
+import { expect as playExpect, test } from '../utility/fixtures';
+import { createKindCluster, deleteKindCluster, ensureKindCliInstalled } from '../utility/operations';
+import { waitForPodmanMachineStartup } from '../utility/wait';
+
+const CLUSTER_NAME: string = 'kind-cluster';
+const CLUSTER_CREATION_TIMEOUT: number = 300_000;
+const KIND_NODE: string = `${CLUSTER_NAME}-control-plane`;
+const KUBERNETES_CONTEXT = `kind-${CLUSTER_NAME}`;
+const KUBERNETES_NAMESPACE = 'default';
+const DEPLOYMENT_NAME = 'test-deployment-resource';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const skipKindInstallation = process.env.SKIP_KIND_INSTALL ? process.env.SKIP_KIND_INSTALL : false;
```

Comment:
please change the statement into `process.env.SKIP_KIND_INSTALL === 'true'` to is will be boolean.